.travis.yml

```diff
@@ -56,10 +56,25 @@ script:
   # step is performed just to shake out data races with concurrent requests.
   - load-generator -config ./test/config/load-generator-config.json > /dev/null
 
+before_deploy:
+  - mkdir -p deploy
+  - cp $(go env GOPATH)/bin/pebble deploy/pebble_linux-amd64
+  - cp $(go env GOPATH)/bin/pebble-challtestsrv deploy/challtestsrv_linux-amd64
```

Comment:
@adferrand I'd prefer to keep the `pebble-` prefix on the `challtestsrv` artifact. Did you have an explicit reason for dropping it that I should consider?

Reply:
No reason in particular, I thought it was a shorter name. I changed for `pebble-challtestsrv`.